features/fedcm.yml

```diff
@@ -4,5 +4,8 @@ spec: https://w3c-fedid.github.io/FedCM/
 group: credential-management
 compat_features:
   - api.CredentialsContainer.get.identity_option
+  - api.CredentialsContainer.get.identity_option.context_option
+  - api.CredentialsContainer.get.identity_option.error_api
```

Comment:
Let's leave this key out (but maybe mention it in a comment above the `compat_features:` line). BCD marked as non-standard because it's not formally part of the spec yet (per https://github.com/w3c-fedid/FedCM/pull/498).